loki_simple.md

```diff
@@ -0,0 +1,252 @@
+# Automatic deployment Loki with the Network Observability Operator
+
+In its current status, deploying and configuring the Loki operator with our NOO
+[might be a complex task](./hack_loki.md). Basically due to the setup and configuration
+of the gateway and the external dependencies towards the Storage Engine. In addition,
```

Comment:
The loki operator gateway need some configuration, but do we need it anyway?
It would be nice to have it so we can query loki from outside, but I am not sure that this is mandatory.

Reply:
I don't think a gateway is our main priority now (unless an architect/product manager says the opposite), but there are some security regulations (I think https://www.fedramp.gov/ does) that enforce secure connections even inside the cluster, especially if your cluster runs in a public cloud.

In a first implementation, users could just manually install their own secured Loki use it from our NOO, but I'd expect that at some point we might want to provide a "zero-click" secured Loki for simple installations.

Reply:
Yes, I understand the needs of having a zero-click Loki installation.

In a previous sync meeting, I remember discussions about using dependcies from the OLM framework. This did not work ?